manifests/params.pp

```diff
@@ -13,6 +13,7 @@
   $custom_repo = false
   $default_package_ensure = installed
   $version = '4.1'
+  $mysql_schema_file = "/usr/share/doc/pdns-backend-mysql-${version}.?/schema.mysql.sql"
```

Comment:
If the user declares the main class with `version` set to `4.0` they'll also need to manually set `mysql_schema_file` to match?

Perhaps, `$mysql_schema_file = "/usr/share/doc/pdns-backend-mysql-4\.*/schema.mysql.sql"` would work fine.

Reply:
Ah, thanks for this. Even after reading your comment I didn't spot the error at first.

Although the attack surface is very small in this case but I try not to use `*` in Puppet. You never know. `4.?.?` may have other issues in the future but we'll cross that bridge when we get there.